CONTRIBUTING.md

```diff
@@ -107,18 +107,18 @@ This repository uses automated linting and automated lint formatting. Use `npm r
 
 To enable linting and formatting during development, install the recommended VS Code extensions. The list of recommended VS Code extensions can be found in `.vscode/extensions.json`.
 
-The default formatter for the workspace should be already configured by `.vscode/settings.json`. To configure it manually go to `File >> Preferences >> Settings >> Workspace >> Text Editor >> Defualt Formatter` and select `Prettier ESLint`. The `Prettier ESLint` option assumes that the recommended VS Code extensions are installed.
+The default formatter for the workspace should be already configured by `.vscode/settings.json`. To configure it manually go to `File >> Preferences >> Settings >> Workspace >> Text Editor >> Default Formatter` and select `Prettier ESLint`. The `Prettier ESLint` option assumes that the recommended VS Code extensions are installed. The workspace configuration file also specifies that the formatter should be run on a file when it is saved. If you want to disable this, the option is `File >> Preferences >> Settings >> Workspace >> Text Editor >> Format On Save`. Or to modify it directly in `.vscode/settings.json`, set `editor.formatOnSave` to `false`.
```

Comment:
Doesn't this result in dirtying the for repo because the workspace seeing is persisted in the vscode folder?

Reply:
It does. Unfortunately what you said about the workspace settings taking precedence over the user settings was correct.

Reply:
Did you search for other alternatives?

This sounds promising: https://github.com/microsoft/vscode/issues/15909#issuecomment-1410351025

Which is referring to:
https://code.visualstudio.com/docs/editor/multi-root-workspaces

Reply:
I had looked at that, but I didn't think we wanted to have our projects appear in two places in the workspace tree:
![image](https://user-images.githubusercontent.com/7282195/233684269-075283dd-ab5a-4429-a9fe-5d3dfe908b63.png)
Maybe we're okay with that? If so, we can turn on "format on save" in the root workspace settings, and users can override this in each individual folder's settings. We would also have to .gitignore the folder-specific settings. I tried creating a settings file in a folder without having it be part of a multi-root workspace, and that didn't work. Would you like me to commit a multi-root workspace file (and the .gitignore changes) to enable folder overrides?

Reply:
Think it should be discussed with the team. I think I advocate for it to be a documented user configuration option instead of adding it to the vscode configuration. If the team prefers to force it on (since user settings dont override workspace settings) then I'll live with it (always have dirty changes locally for disabling the workspace setting).

Reply:
Changed to just documenting the option in our CONTRIBUTING file.